docs/content/dagster-cloud/deployment/serverless.mdx

```diff
@@ -149,3 +149,62 @@ Unlike Hybrid, Serverless Deployments on Dagster Cloud require direct access to
 - Secrets and source code are built into the image directly. Images are stored in a per-customer container registry with restricted access.
 - User code is securely sandboxed using modern container sandboxing techniques.
 - All production access is governed by industry-standard best practices which are regularly audited.
+
+---
+
+## Run isolation
```

Comment:
I think we should clarify that isolation is relative to your own resources - that you're never actually sharing compute environments with anybody outside your organization.